ddtrace/contrib/django/utils.py

```diff
@@ -260,7 +263,11 @@ def _after_request_tags(pin, span, request, response):
                 if name:
                     request_headers[name] = value
 
-        response_headers = dict(response.items())
+        # DEV: Resolve the view and resource name at the end of the request in case
+        #      urlconf changes at any point during the request
+        _set_resolver_tags(pin, span, request)
+
+        response_headers = dict(response.items()) if response else {}
```

Comment:
(aside) I wonder if we actually have to make a copy 🤔

Reply:
good question, taking this as it was, so don't want to adjust here.

Reply:
```suggestion
        response_headers = response.copy() if response else {}
```
![Screenshot 2021-07-30 at 09 28 12](https://user-images.githubusercontent.com/20231758/127624957-7d03425b-58a4-438c-b8a3-61ad00e2f091.png)

Reply:
We actually copy in set_http_meta already: https://github.com/DataDog/dd-trace-py/blob/91a170ad4dfd2ae9d346909d2efe104c554ed91b/ddtrace/contrib/trace_utils.py#L270